cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/store/RepositorySourceControlMetadataStoreTest.java

```diff
@@ -33,8 +45,239 @@ public abstract class RepositorySourceControlMetadataStoreTest {
   private static final ApplicationReference APP_REF = new ApplicationReference(NAMESPACE, NAME);
   private static final Boolean IS_SYNCED = true;
 
+  private static final String TYPE = EntityType.APPLICATION.toString();
+
   protected static TransactionRunner transactionRunner;
 
+  @Test
+  public void testScanRepositorySourceControlMeta() {
```

Comment:
divide into seperatev tests for normal/sort/filter

Reply:
sure, done